commands/migratelxc.go

```diff
@@ -51,6 +67,16 @@ func (c *migrateLXCCommand) Init(args []string) error {
 	return cmd.CheckEmpty(args)
 }
 
+func (c *migrateLXCCommand) Run(ctx *cmd.Context) error {
+	if c.match != "" {
+		c.extraOptions = append(c.extraOptions, "--match="+c.match)
```

Comment:
Any quoting considerations here? Presumably this works ok with a `.*` pattern, maybe it's not a problem.

Reply:
Yeah I guess we should, I was trying not to overdo it. We've got a util function for quoting, I'll stick it in there.